paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUITest.kt

```diff
@@ -79,6 +83,29 @@ internal class PaymentMethodVerticalLayoutUITest {
         composeRule.onNodeWithTag(TEST_TAG_NEW_PAYMENT_METHOD_ROW_BUTTON + "_card").assertExists()
         composeRule.onNodeWithTag(TEST_TAG_NEW_PAYMENT_METHOD_ROW_BUTTON + "_cashapp").assertExists()
         composeRule.onNodeWithTag(TEST_TAG_NEW_PAYMENT_METHOD_ROW_BUTTON + "_klarna").assertExists()
+
+        composeRule.onNodeWithTag(
+            TEST_TAG_SAVED_PAYMENT_METHOD_ROW_BUTTON + "_${PaymentMethodFixtures.displayableCard().paymentMethod.id}"
+        ).assertExists()
+    }
+
+    @Test
+    fun savedPaymentMethodIsSelected_whenSelectedIndexIsSavedPmIndex() = runScenario(
```

Comment:
Should we assert the selected one is the correct one, and that none of the "new" LPMs are selected?

Reply:
Yup! That's what I meant.